src/meta/app/src/principal/file_format.rs

```diff
@@ -689,50 +703,57 @@ impl Display for FileFormatParams {
     }
 }
 
-pub fn check_str_len(
-    option: &str,
-    min: usize,
-    max: usize,
-    fmt_name: &str,
-    option_name: &str,
-) -> Result<()> {
+pub fn check_row_tag(option: &str) -> std::result::Result<(), String> {
+    let len = option.as_bytes().len();
+    let (max, min) = (1024, 1);
+    if len > max || len < min {
+        Err(format!("length must in [{min}, {max}]"))
+    } else {
+        Ok(())
+    }
+}
+
+pub fn check_field_delimiter(option: &str) -> std::result::Result<(), String> {
     let len = option.as_bytes().len();
-    if len < min || len > max {
-        Err(ErrorCode::InvalidArgument(format!(
-            "len of option {option_name} for {fmt_name} must in [{min}, {max}], got {option}"
-        )))
+    if len != 1 {
+        Err("only accept one char".into())
+    } else if option.as_bytes()[0].is_ascii_alphanumeric() {
+        return Err("not accept alphanumeric".into());
```

Comment:
Please make the error message more friendly to user, guide them to do right.

Reply:
Others same, reference:
https://github.com/datafuselabs/databend/pull/14133/files

Reply:
please review again